packages/oui-tooltip/tooltip.less

```diff
@@ -0,0 +1,24 @@
+@import './_variables';
+
+.oui-tooltip {
+  min-width: @oui-tooltip-min-width;
```

Comment:
Put this inside a mixin, we want to keep root file as simple as possible. We want other dev to be able to recreate any component using mixins.

Reply:
Ok, done